app/controllers/application_controller.rb

```diff
@@ -7,6 +7,6 @@ def current_user
   end
 
   def authenticate_user!
-    redirect_to new_user_session_path, flash: { error: "You need to sign in for that!!" } if session[:user_id].blank?
+    redirect_to new_session_path, flash: { error: "You need to sign in for that!!" } if session[:user_id].blank?
```

Comment:
it will be good to have rubocop as part of ur test suite and setting a fixed set of rules for a project helps when more ppl are involved to maintain uniformity. U can enforce stuff like line length, line count etc.
Internationalisation is a good practise to have in the longer run..ur controllers are clean, u can reuse messages.
Stuff like rubocop and using translations just make ur code more readable